components/HelloWorld.vue

```diff
@@ -1,7 +1,185 @@
 <template>
-  <div>Hello world!</div>
+  <div>
```

Comment:
I would move this all to a new component in `components/Navbar.vue`. Then you can just use `<Navbar />` in `index.vue`.